A3A/addons/core/functions/Missions/fn_effects

```diff
@@ -0,0 +1,84 @@
+/*
+    File: fn_effects.sqf
+    Author: unknown
+    Adopted: wersal
+
+    Description:
+        creates a visual effect of an crashing object
+
+    Parameter(s):
+        _object - vehicle or a prop to apply effects to
+
+*/
+#include "..\..\script_component.hpp"
+FIX_LINE_NUMBERS()
+
+params ["_object"];
+
+private _posATL = _object modelToWorld [0,0,0];
+
+private _ps0 = "#particlesource" createVehicleLocal _posATL;
+_ps0 setParticleParams [
+["\A3\Data_F\ParticleEffects\Universal\Universal", 16, 10, 32], "", "Billboard",
+0, 3, [0, 0, 0.25], [0, 0, 0.5], 1, 1, 0.9, 0.3, [3.5],
+[[1,1,1, 0.0], [1,1,1, 0.3], [1,1,1, 0.0]],
+[0.75], 0, 0, "", "", _ps0, rad -45];
+_ps0 setParticleRandom [0.2, [1, 1, 0], [0.5, 0.5, 0], 0, 0.5, [0, 0, 0, 0], 0, 0];
+_ps0 setDropInterval 0.03;
+
+private _ps1 = "#particlesource" createVehicleLocal _posATL;
+_ps1 setParticleParams [
+["\A3\Data_F\ParticleEffects\Universal\Universal", 16, 7, 16, 1], "", "Billboard",
+1, 30, [0, 0, 0], [0, 0, 4.5], 0, 10, 7.9, 0.5, [8, 24, 30],
+[[0.2, 0.2, 0.2, 0], [0.2, 0.2, 0.2, 0.3], [0.2, 0.2, 0.2, 0.3], [0.35, 0.35, 0.35, 0.2], [0.5, 0.5, 0.5, 0]],
+[0.25], 1, 0, "", "", _ps1];
+_ps1 setParticleRandom [0, [0.4, 0.4, 0], [0.4, 0.4, 0], 0, 0.25, [0, 0, 0, 0.1], 0, 0];
+_ps1 setDropInterval 0.02;
+
+private _ps2 = "#particlesource" createVehicleLocal _posATL;
+_ps2 setParticleParams [
+["\A3\Data_F\ParticleEffects\Universal\Universal", 16, 9, 16, 0], "", "Billboard",
+1, 30, [0, 0, 0], [0, 0, 4.5], 0, 10, 7.9, 0.5, [8, 24, 30],
+[[0.33, 0.33, 0.33, 0], [0.33, 0.33, 0.33, 0.8], [0.33, 0.33, 0.33, 0.8], [0.66, 0.66, 0.66, 0.4], [1, 1, 1, 0]],
+[0.25], 1, 0, "", "", _ps2];
+_ps2 setParticleRandom [0, [0.4, 0.4, 0], [0.4, 0.4, 0], 0, 0.25, [0, 0, 0, 0.1], 0, 0];
+_ps2 setDropInterval 0.02;
+
+_ps0 attachTo [_object, [0, 0, 1]];
+_ps1 attachTo [_object, [0, 0, 1]];
+_ps2 attachTo [_object, [0, 0, 1]];
+
+sleep 11;
+
+if (typeOf _object == "SpaceshipCapsule_01_F") then {
+	
+	setAperture 10;
+
+	playSound3D ["x\A3A\addons\core\Sounds\Misc\Sonic.ogg", _object, false, getPosASL _object, 5, 1, 5000];
```

Comment:
You should be able to use QPATHTOFOLDER since you #include script_component.hpp at the top
```suggestion
	playSound3D [QPATHTOFOLDER(Sounds\Misc\Sonic.ogg), _object, false, getPosASL _object, 5, 1, 5000];
```